### ISSUE TITLE
Incorrect output for multiline variant group

### ISSUE THREAD

**Issue by vjee** [windicss/windicss#468]
When defining a variant group over multiple lines, the name of the variant is repeated many times in the output, though the correct classes are still generated so technically it still works.

```html
<div class="
  w-8
  h-8
  bg-red-500
  hover:(
  w-16
  h-16
  bg-green-500)
"></div>
```

> Note: This is the type of output `prettier` will give you if there are enough utilities in the variant group.

Output:
```
"
      w-8
      h-8
      bg-red-500
      hover: hover: hover: hover: hover: hover: hover: hover:w-16 hover: hover: hover: hover: hover: hover: hover:h-16 hover: hover: hover: hover: hover: hover: hover:bg-green-500
    "
```

> Note: The output above is from Vite+Vue3.
> Trying it in the Windi CSS playground does not work (styles are not generated for the variant group).
> Presumably because of different whitespace handling.
> https://windicss.org/play.html?html=DwEwlgbgBAxgNgQwM5ILwCIBQUoHcC0AHNlABZEkBGA5vgE4CmI%2BArAAxsmkD2EDdALgAUJAgEYAbF3ySqtaowYA7VhwCUmdAD5gAenAQtQA&css=ARA

**Comment by alexanderniebuhr** [comment]
@vjee thanks for your report. Do you use our [vite-plugin](https://github.com/windicss/vite-plugin-windicss) ?

/cc @antfu

**Comment by vjee** [comment]
@alexanderniebuhr

Indeed. I have created a reproduction at https://github.com/vjee/vite-vue3-windi.
While creating this repro, I have noticed this bug is caused by the _leading spaces_ of multiline variant groups.

If there are _no_ leading spaces everything works fine:
```js
const className = `
dark:(bg-gray-800
text-gray-300)
`;

// dark:bg-gray-800 dark:text-gray-300
```

If there _are_ leading spaces, then `${variant}:` will be added to the result once for every leading space:
```js
const className = `
    dark:(bg-gray-800
    text-gray-300)
`;

// dark:bg-gray-800 dark: dark: dark: dark: dark:text-gray-300
```

If not all newlines have the same amount of leading spaces, the amount of `${variant}:`'s generated equals the smallest amount of spaces:
```js
const className = `
      dark:(bg-gray-800
  text-gray-300)
`;

// dark:bg-gray-800 dark: dark: dark:text-gray-300
```

When defining the class in the template, the case above does not apply.
```html
<template>
  <div class="
        dark:(bg-gray-800
  text-gray-300)
  "></div>
</template>

<!-- dark:bg-gray-800 dark: dark: dark: dark: dark: dark: dark: dark:dark:text-gray-300 -->
```

Hopefully this can help.
